apps/foundation-demo/src/app/checkout/components/shipping-async-wrapper/shipping-async-wrapper.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="checkout__shipping">
```

Comment:
What's the purpose of this component? It's not clear to me off the bat, and probably requires further documentation.

Reply:
I think I originally put it in there so I didn't have any shipping logic in the checkout page itself, but then I also added the shipping-async-wrapper. So this component is useless now. I removed it